pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1056,13 +1063,13 @@ func (b *MasqueradeBindMechanism) createNatRulesUsingIptables(protocol iptables.
 	return nil
 }
 
-func (b *MasqueradeBindMechanism) skipForwardingForPortsUsingIptables(protocol iptables.Protocol, reservedPorts []string) error {
+func (b *MasqueradeBindMechanism) skipForwardingForPortsUsingIptables(protocol iptables.Protocol, ports []string) error {
```

Comment:
Unless there's something I'm missing here, you've introduced this in the previous commit; might as well name it accurately the first time.

Reply:
Thanks, done